### PR TITLE
integration: fix a data race about `i` and `tt` in TestV3WatchFromCur…

### PR DESCRIPTION
**Pull request by yuzeming**
…rentRevision

don't references to loop variables from within go anonymous function

Fixes etcd-io#10899

### CODE REVIEW

File: integration/v3_watch_test.go [hunk 1]
```diff
@@ -286,6 +288,8 @@ func TestV3WatchFromCurrentRevision(t *testing.T) {
 			t.Errorf("unexpected pb.WatchResponse is received %+v", nr)
 		}
 
+		<-ch
```
Comment (jingyih) [inline integration/v3_watch_test.go]: Add a comment here? Something like "wait for the client to finish sending the keys before terminating the cluster".

**Comment by jingyih** [comment]
Could you change the commit title? (maybe squash the commits?)

https://travis-ci.com/etcd-io/etcd/jobs/217434454#L582

**Comment by yuzeming** [comment]
Now it should be fine, thanks.

**Comment by jingyih** [comment]
lgtm

**Pull request merged**